src/components/Profile/components/PersonalInfoList/components/UpdatePhoneDialog/index.scss

```diff
@@ -0,0 +1,30 @@
+@import '../../../../../../vars';
```

Comment:
This might be better and cleaner to reference using `/` to start at the root directory of the app, but it's up to you.

Reply:
Unfortunately, `scss` files don't allow relative imports. This is consistent with our other styling files.

Reply:
Everywhere use `'../../vars';` so I'll just leave like this.